kernel/src/cpuid.c

```diff
@@ -0,0 +1,38 @@
+#include <stdint.h>
+
+#include "io.h"
+#include "cpuid.h"
+
+void print_reg(uint32_t r) {
```

Comment:
Missing const qualifier
```suggestion
void print_reg(const uint32_t r) {
```